From 4dd641efa81998ed328fd2c74be8beee7aa32dad Mon Sep 17 00:00:00 2001
From: =?UTF-8?q?Zbigniew=20J=C4=99drzejewski-Szmek?= <zbyszek@in.waw.pl>
Date: Fri, 9 May 2014 08:39:25 -0400
Subject: [PATCH] man: sd_journal_send does nothing when journald is not
 available

https://bugzilla.redhat.com/show_bug.cgi?id=1096067

(cherry-picked from bdf9fc1a940e342afb7a78075984419cb3bc3135)

Resolves: #1147524
---
 man/sd_journal_print.xml | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/man/sd_journal_print.xml b/man/sd_journal_print.xml
index a716cc3..d934b0a 100644
--- a/man/sd_journal_print.xml
+++ b/man/sd_journal_print.xml
@@ -218,6 +218,11 @@ sd_journal_send("MESSAGE=Hello World, this is PID %lu!", (unsigned long) getpid(
                 errno-style error code. The
                 <citerefentry><refentrytitle>errno</refentrytitle><manvolnum>3</manvolnum></citerefentry>
                 variable itself is not altered.</para>
+
+                <para>If
+                <citerefentry><refentrytitle>systemd-journald</refentrytitle><manvolnum>8</manvolnum></citerefentry>
+                is not running (the socket is not present), those
+                functions do nothing, and also return 0.</para>
         </refsect1>
 
         <refsect1>
